README.md

```diff
@@ -188,6 +188,30 @@ Returns
 """
 ```
 
+### Deployment
+
+When deploying your project that utilizes our CLI tool, it's essential to ensure that all necessary dependencies are included in your project's environment. Depending on your project setup, add these dependencies either through a requirements.txt file or using Poetry.
+
+#### requirements.txt
+
+```txt
+requirements.txt
+
+ecdsa==0.19.0
+httpx==0.27.0
+pydantic==2.7.4
+pydantic_core==2.18.4
+requests==2.28.2
+```
+
+#### poetry
+
+If you are managing your project with Poetry, execute the following command to add these dependencies to your pyproject.toml file:
+
+```shell
+poetry add ecdsa==0.19.0 httpx==0.27.0 pydantic==2.7.4 pydantic_core==2.18.4 requests==2.28.2
+```
```

Comment:
I don't think there's a need for requirements.txt if dependencies are being managed with poetry.

You can also add `pip install -r requirements.txt` as an alternative command if the user manages dependencies with pip.

Reply:
@Bashorun97 thanks for the review. I added the Poetry instructions for situations where users of the package are using Poetry for their dependency management.  This way they know what packages they need to install in their project after generating the Sauron files.